cpp/src/parquet/bloom_filter.h

```diff
@@ -167,6 +167,12 @@ class PARQUET_EXPORT BloomFilter {
 
   virtual ~BloomFilter() = default;
 
+  // Variant of const pointer argument to facilitate template
```

Comment:
@emkornfield I think pointer is ok here, would you think 

```c++
  virtual void Hash(const FLBA& value, uint32_t type_len) const;
  virtual void Hash(const ByteArray& value, uint32_t type_len) const;
  virtual void Hash(const Int96& value, uint32_t type_len) const;
```

The code above is better?